pkg/bff/client.go

```diff
@@ -71,6 +88,100 @@ func (c *GDSClient) ConnectMembers(conf config.MembersConfig, opts ...grpc.DialO
+	return c.admin.Authenticate(ctx, in)
+}
+
+func (c *GDSClient) Reauthenticate(ctx context.Context, in *admin.AuthRequest) (out *admin.AuthReply, err error) {
+	return c.admin.Reauthenticate(ctx, in)
+}
+
+func (c *GDSClient) AdminSummary(ctx context.Context) (out *admin.SummaryReply, err error) {
+	return c.admin.AdminSummary(ctx)
+}
+
+func (c *GDSClient) Autocomplete(ctx context.Context) (out *admin.AutocompleteReply, err error) {
+	return c.admin.Autocomplete(ctx)
+}
+
+func (c *GDSClient) ReviewTimeline(ctx context.Context, params *admin.ReviewTimelineParams) (out *admin.ReviewTimelineReply, err error) {
+	return c.admin.ReviewTimeline(ctx, params)
+}
+
+func (c *GDSClient) ListVASPs(ctx context.Context, params *admin.ListVASPsParams) (out *admin.ListVASPsReply, err error) {
+	return c.admin.ListVASPs(ctx, params)
+}
+
+func (c *GDSClient) RetrieveVASP(ctx context.Context, id string) (out *admin.RetrieveVASPReply, err error) {
+	return c.admin.RetrieveVASP(ctx, id)
+}
+
+func (c *GDSClient) UpdateVASP(ctx context.Context, in *admin.UpdateVASPRequest) (out *admin.UpdateVASPReply, err error) {
+	return c.admin.UpdateVASP(ctx, in)
+}
+
+func (c *GDSClient) DeleteVASP(ctx context.Context, id string) (out *admin.Reply, err error) {
+	return c.admin.DeleteVASP(ctx, id)
+}
+
+func (c *GDSClient) ListCertificates(ctx context.Context, vaspID string) (out *admin.ListCertificatesReply, err error) {
+	return c.admin.ListCertificates(ctx, vaspID)
+}
+
+func (c *GDSClient) ReplaceContact(ctx context.Context, in *admin.ReplaceContactRequest) (out *admin.Reply, err error) {
+	return c.admin.ReplaceContact(ctx, in)
+}
+
+func (c *GDSClient) DeleteContact(ctx context.Context, vaspID string, kind string) (out *admin.Reply, err error) {
+	return c.admin.DeleteContact(ctx, vaspID, kind)
+}
+
+func (c *GDSClient) CreateReviewNote(ctx context.Context, in *admin.ModifyReviewNoteRequest) (out *admin.ReviewNote, err error) {
+	return c.admin.CreateReviewNote(ctx, in)
+}
+
+func (c *GDSClient) ListReviewNotes(ctx context.Context, id string) (out *admin.ListReviewNotesReply, err error) {
+	return c.admin.ListReviewNotes(ctx, id)
+}
+
+func (c *GDSClient) UpdateReviewNote(ctx context.Context, in *admin.ModifyReviewNoteRequest) (out *admin.ReviewNote, err error) {
+	return c.admin.UpdateReviewNote(ctx, in)
+}
+
+func (c *GDSClient) DeleteReviewNote(ctx context.Context, vaspID string, noteID string) (out *admin.Reply, err error) {
+	return c.admin.DeleteReviewNote(ctx, vaspID, noteID)
+}
+
+func (c *GDSClient) ReviewToken(ctx context.Context, vaspID string) (out *admin.ReviewTokenReply, err error) {
+	return c.admin.ReviewToken(ctx, vaspID)
+}
+
+func (c *GDSClient) Review(ctx context.Context, in *admin.ReviewRequest) (out *admin.ReviewReply, err error) {
+	return c.admin.Review(ctx, in)
+}
+
+func (c *GDSClient) Resend(ctx context.Context, in *admin.ResendRequest) (out *admin.ResendReply, err error) {
+	return c.admin.Resend(ctx, in)
+}
+
+// GDS methods
```

Comment:
Because the `APIv2` struct is exported, I think these methods can be added by direct embedding rather than call throughs.

Reply:
I think I understand what you mean by embedding the `APIv2` struct into the `GDSClient`, but the fields in `APIv2` are not exported so it doesn't look like I can set them?